FlightOps/template/FIRST.html

```diff
@@ -1,12 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
 </head>
+
 <body>
   PCS3642 - Laboratório de Engenharia de Software I, também conhecida como a MELHOR matéria da USP 🥇
 </body>
-</html>
+
+</html>
```

Comment:
```suggestion
</html>

```
bom deixar essa linha